tests/kiwi_images_test/validate_local_kiwi_image.pm

```diff
@@ -0,0 +1,25 @@
+# SUSE's openQA tests
+#
+# Copyright 2019 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Validate the locally built Kiwi qcow2 image by booting it.
+# Maintainer:  QE Core <qe-core@suse.de>
+
+use base "installbasetest";
+use strict;
+use warnings;
+use testapi;
+
+sub run {
+    my $args_gw = testapi::host_ip();
+    # login in to system
+    enter_cmd("root");
+    sleep(2);
```

Comment:
I fear that the sleep will need to be changed in the future, can you add a comment to rework the test if it fails because of this? as 2 seconds might be too little if the host system is under heavy load

Reply:
I removed the sleep in the code by handling the login to the system via a function call, `select_serial_terminal`(). Additionally, I modified the user accounts in the Kiwi XML file with the default passwords.